selfdrive/ui/paint.cc

```diff
@@ -670,18 +663,33 @@ static void ui_draw_vision_footer(UIState *s) {
 #endif
 }
 
-void ui_draw_vision_alert(UIState *s, int va_size, int va_color,
+void ui_draw_vision_alert(UIState *s, cereal::ControlsState::AlertSize va_size, int va_color,
                           const char* va_text1, const char* va_text2) {
   const UIScene *scene = &s->scene;
   const bool hasSidebar = !scene->uilayout_sidebarcollapsed;
   const bool mapEnabled = scene->uilayout_mapenabled;
   bool longAlert1 = strlen(va_text1) > 15;
 
   const uint8_t *color = alert_colors[va_color];
-  const int alr_s = alert_sizes[va_size];
+  int alr_s = 0;
```

Comment:
I find this much less readable. Does

const int alert_sizes[] = {		
   [cereal::ControlsState::AlertSize::NONE] = 0,		
   [cereal::ControlsState::AlertSize::SMALL] = 241,

work?

Reply:
no, it didn't work in this case:
error: integral constant expression must have integral or unscoped enumeration type, not 'capnp::schemas::ThermalStatus_98c15adf4549151f'